bgpd/bgp_routemap.c

```diff
@@ -4064,6 +4088,29 @@ DEFUN (set_ip_nexthop_unchanged,
 		    "unchanged");
 }
 
+DEFUN (set_distance,
+       set_distance_cmd,
+       "set distance (0-255)",
```

Comment:
`set distance 0` - means use native distance selection method?

Reply:
According to the doc, it means system, kernel, or connected. I'd guess setting distance to 0 here would mark it as connected? At any rate, the 0 distance is documented here --

http://docs.frrouting.org/en/latest/zebra.html?highlight=distance#administrative-distance

So I don't think it needs to be mentioned in this change (?).